test/testers.jl

```diff
@@ -551,7 +551,7 @@ end
         f_noninferrable(x) = Ref{Real}(x)[]
 
         @test @maybe_inferred(identity(1)) == 1
-        errors("not inferrable") do
+       @test errors("not inferrable") do
```

Comment:
ooops this is prob me not counting my indenting right
```suggestion
        @test errors("not inferrable") do
```